CaptureSDK/GiniCaptureSDK/Documentation/source/Migration to version 3.x.x.md

```diff
@@ -1,8 +1,178 @@
 Migrate from the Gini Capture SDK 1.x.x to 3.x.x
 =================================================
 
-We've removed the Component API integration option
-
-Introduced centralized customization for the color pallete, typograhy.
+In version 3.0.0 we modernized our UI. In addition, we simplified how the UI
+is customized, introduced centralized customization for the color pallete, typograhy.
+We also removed the Component API integration option and unified the public API of the SDK and introduced an easier way to customize certain parts of the UI.
 
 Please, find more details in [Getting started](https://developer.gini.net/gini-mobile-ios/GiniCaptureSDK/getting-started.html).
+
+# Migrate from Compomemt API
+
+The Component API allowed more UI customization options at the cost of a more difficult integration and maintenance. It
+was based on the view controllers, and you had to manage navigation between them and also update the navigation whenever we introduced
+breaking changes.
+
+Maintaining the Component API along with the simpler Screen API required an increasing amount of effort as we added new
+features. We decided therefore to unify both APIs and introduce the ability to inject fully custom UI elements.
+
+The major benefit of the Component API was the ability to use a custom navigation bar. Via
+`GiniConfiguration.shared.customNavigationController` that is still possible with the new public API.
+
+The following steps will help you migrate to the new public API:
+
+* Configure the SDK the same way as before by using `GiniConfiguration`.
```

Comment:
[Suggestion] Maybe we could mention that this is a singleton now, if they were used to create a new instance of it before?

Reply:
Good point. 👍🏻 I'll add it.